plugins/com.google.cloud.tools.eclipse.usagetracker/src/com/google/cloud/tools/eclipse/usagetracker/AnalyticsPingManager.java

```diff
@@ -251,6 +252,9 @@ static String getParametersString(Map<String, String> parametersMap) {
     return resultBuilder.toString();
   }
 
+  // To prevent showing multiple opt-in dialogs. See OptInDialog#open() and OptInDialog#close().
```

Comment:
Another thing to note here is that we treat dialog-close as opt-out rather than unknown.  So we have an answer once we've shown the dialog once and thus don't need to restore the dialogOpenStatus to false.  Maybe `dialogOpenedStatus`?